pkgs/development/python-modules/numexpr/default.nix

```diff
@@ -16,16 +15,11 @@ buildPythonPackage rec {
   };
 
   # Remove existing site.cfg, use the one we built for numpy.
-  # Somehow openmp needs to be added to LD_LIBRARY_PATH
-  # https://software.intel.com/en-us/forums/intel-system-studio/topic/611682
   preBuild = ''
     rm site.cfg
     ln -s ${numpy.cfg} site.cfg
-    export LD_LIBRARY_PATH=${llvmPackages.openmp}/lib
```

Comment:
This kind of stuff worked fine during the nix build and in nix shells, but fails immediately if you try to install and use it; this whole package as a whole has gotten significantly simpler now that we provide a sane default and just stick with Intel's recommendation to use a runtime LD_PRELOAD if you want something else, since it's not used at build-time and is binary compatible with the LLVM openmp.